### PR TITLE
[F] Status responds with index information.

### PR DESCRIPTION
**Pull request by Xhark-Blues**

### CODE REVIEW

File: src/auto-redis/status.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}) {
+  return async function () {
+    const lists = await pzkeys('*');
+    const res = await all(lists.map( async (zset:string) => {
+      const length = await pzcard(zset);
+      const words = await pzrange(zset, 0, -1);
+      const filtered = words.filter( (word:string) =>
+        word.charAt(word.length -1) === '*'
+      );
+      console.log(filtered);
```
Comment (Istar-Eldritch) [inline src/auto-redis/status.ts]: Remote this log please

File: src/service.ts [hunk 2]
```diff
@@ -25,25 +25,47 @@ export default async function init({
...
-
   const datasetProperties = [
     'disease',
     'assay',
     'tissue',
     'technology'
   ];
 
+  iris.register<any, any>({
+    pattern: 'status.muninn.v2',
```
Comment (Istar-Eldritch) [inline src/service.ts]: I would keep as pattern the `status.muninn` directly.

Comment (Istar-Eldritch) [reply src/service.ts]: What was your point with adding the `v2` there? Keep the old interface alive?

**Review by Istar-Eldritch — CHANGES_REQUESTED** [review]

**Comment by Istar-Eldritch** [comment]
Just a few questions I would like you to answer and minor changes.

File: src/service.ts (hunk 2 above)
Comment (Xhark-Blues) [reply src/service.ts]: Yes, basically. Should I overwrite it?

Comment (Istar-Eldritch) [reply src/service.ts]: I would say so, no one depends on it.

**Review by Istar-Eldritch — APPROVED** [review]
:rocket:

**Comment by Istar-Eldritch** [comment]
Remember to use the name conventions for merging!

**Pull request merged**